presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -83,4 +94,31 @@ public BaseJdbcConfig setJdbcTypesMappedToVarchar(String jdbcTypesMappedToVarcha
         this.jdbcTypesMappedToVarchar = ImmutableSet.copyOf(Splitter.on(",").omitEmptyStrings().trimResults().split(nullToEmpty(jdbcTypesMappedToVarchar)));
         return this;
     }
+
+    @Min(0)
+    @Max(38)
+    public Integer getDecimalDefaultScale()
+    {
+        return decimalDefaultScale;
+    }
+
+    @Config("decimal-default-scale")
```

Comment:
I don't think it is a generic property. I think that is connector specific thing. Decimal semantic differs between RDBMSes.

decimal.default-scale?

Reply:
Each catalog has it's own property and it appears as `catalog_name.decimal_default-scale`

Reply:
`decimal-default-scale` is a catalog session parameter presented as `catalog_name. decimal_default_scale`